pkgs/test_core/lib/backend.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+export 'package:test_api/src/backend/metadata.dart' show Metadata;
```

Comment:
should this be `export 'package:test_api/backend.dart' show ...;`?

Reply:
done